easybuild/easyblocks/e/elpa.py

```diff
@@ -30,16 +30,20 @@
 @author: Kenneth Hoste (Ghent University)
 """
 import os
+from distutils.version import LooseVersion
 
+import easybuild.tools.environment as env
```

Comment:
and move this one down a few lines to maintain alphabetical order

Reply:
Don't we do full imports before `from ... import ...`? I wasn't sure, but looked at another easyblock that I believe had this order (don't remember which one though...)

Reply:
See e.g. https://github.com/easybuilders/easybuild-easyblocks/blob/e23db61e05f4dbc408351ccec13952a987eeeeea/easybuild/easyblocks/n/netcdf_fortran.py

Btw, I guess this is one of the things we might want to document about code style in framework/blocks at https://github.com/easybuilders/easybuild/wiki/Review-process-for-contributions :)

Reply:
yeah I overlooked that it was a full import